### DISCUSSION TITLE
add gpt-4o-2024-08-06 model to langfuse models list

### DISCUSSION THREAD

**Discussion by alex-stoica** [langfuse/langfuse#3875] [Ideas]
### Describe the feature or potential improvement

Please add the `gpt-4o-2024-08-06` model to the list of models in Langfuse. It currently seems to be missing from the Models page.

The corresponding documentation for this model can be found at:
https://platform.openai.com/docs/models/gpt-4o

**Comment by hassiebp** [comment]
Hi @alex-stoica - that model should be supported already. Are you self hosted? If so, is your Langfuse version up to date? I can see the `gpt-4o-2024-08-06` on page 2 of the models page in the cloud version.

**Reply by alex-stoica** [reply]
Hi @hassiebp - yes, self-hosted. I can only see `gpt-4o-2024-05-13`  and `gpt-4o` in the "Tracing / Models" page. I have v2.59.1, maybe I need an upgrade, indeed. Thanks!

**Reply by hassiebp** [reply]
Yes, upgrading should do the trick :)
